config-editor/config-editor-ui/src/app/components/testing/test-case-editor/test-case-editor.component.ts

```diff
@@ -33,6 +33,26 @@ export class TestCaseEditorComponent implements OnInit, OnDestroy {
   testStoreService: TestStoreService;
   form: FormGroup = new FormGroup({});
   private markHistoryChange = false;
+  private testConfigSpec: TestConfigSpec = undefined;
+  private TEST_CASE_TESTER_KEY = "test_case_tester";
+  testCaseConfigTestersFields: FormlyFieldConfig[] = [ 
```

Comment:
a lot of repeated code for the dropdow, maybe could create a generic component for this put in both test-case and config-editor?